lotus/sem_ops/sem_extract.py

```diff
@@ -11,26 +12,29 @@
 
 def sem_extract(
     docs: list[str],
-    model: lotus.models.LM,
-    user_instruction: str,
+    model: LM,
+    columns: list[str],
+    extract_quotes: bool = True,
     postprocessor: Callable[[list[str]], SemanticExtractPostprocessOutput] = extract_postprocess,
 ) -> SemanticExtractOutput:
     """
-    Extracts from a list of documents using a model.
+    Schemas a list of documents using a model.
 
     Args:
-        docs (list[str]): The list of documents to extract from.
+        docs (list[str]): The list of documents to schema.
         model (lotus.models.LM): The model to use.
-        user_instruction (str): The user instruction for extract.
-        postprocessor (Callable): The postprocessor for the model outputs. Defaults to extract_postprocess.
+        columns (list[str]): The columns to schema.
+        extract_quotes (bool, optional): Whether to extract quotes for user_instruction. Defaults to True.
+        postprocessor (Callable): The postprocessor for the model outputs. Defaults to to_schema_postprocess.
```

Comment:
`to_schema_postprocess` no longer exists - change the description to `extract_postprocess`